docs/triggers.md

```diff
@@ -0,0 +1,34 @@
+<!--
+---
+linkTitle: "Trigger Templates"
+weight: 9
+---
+-->
+# Triggers
+
+A `Trigger` is resource that combines `TriggerTemplate`, `TriggerBindings` and `interceptors`. The `Trigger` is processed by EventListener which referenced it when it receives an incoming.
```

Comment:
I know the docs for bindings/template don't do this yet, but it will be nice for the docs to have some standard headings i.e. at least a standard  `syntax` heading: https://github.com/tektoncd/triggers/blob/master/docs/eventlisteners.md#syntax